zinc-compile/src/test/scala/inc/DocSpec.scala

```diff
@@ -16,7 +16,7 @@ class DocSpec extends UnitSpec {
     IO.withTemporaryDirectory { cacheDir =>
       IO.withTemporaryDirectory { out =>
         val javadoc = Doc.cachedJavadoc("Foo", cacheDir, local)
-        javadoc.run(List(knownSampleGoodFile), Nil, out, Nil, log, reporter)
+        javadoc.run(List(knownSampleGoodFile), Nil, out, Nil, null, log, reporter)
```

Comment:
Fixed by introducing `NoopClassFileManager`, since it's used only by test, placed it under `zinc-compile-core/../test`. Similarly `CollectingClassFileManager` is also used only for testing.